vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    state: present
+
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
+    home: /home/minio
+
+- name: create minio bin directory
+  become: yes
+  file:
+    path: "{{root_folder}}/{{ item }}/minio/"
+    state: directory
+    owner: minio
+  with_items:
+    - "app"
+    - "data"
+    - "log"
+    - "tmp"
+
+- name: download minio file from internet
```

Comment:
Ah this looks very nice now :-) the only problem is the `download.yml` doesn't support `bin_t`.  Can you now set `bin_t` mode in your `file` task, below this one?  Then it should be good to go.